op-service/txmgr/txmgr.go

```diff
@@ -112,15 +120,32 @@ func (m *SimpleTxManager) IncreaseGasPrice(ctx context.Context, tx *types.Transa
 		gasTipCap = tip
 	}
 
+	// new = old * (100 + priceBump) / 100
```

Comment:
The `IncreaseGasPrice` method is called in a loop, with a 1 second interval in testing, and an unchecked but explicitly configured flag/env var in prod. To not burn ourselves when we accidentally set this interval too low, we should try to back-off the gas increases with longer and longer intervals, to avoid rapid increases. Also see other related discussion in Slack.


Raising this now because the minimum gas price increase makes the increases go higher exponentially, it doesn't level off anymore.

Reply:
Well the other option is to not bump the gas price at all, but return the previous transaction & resubmit it. My sense is that @tynes wanted to always do the price bump. I guess there's also the difference between the new price is less than the old price & the new price is in between the old price in the threshold - in this case we may want to bump, but if it's below the old price, we probably want to not bump.

Reply:
Ok, I added a commit to fix this.